File: sum_vectors.py
def main():
    a = [n for n in range(1, 10_000_001)]
    b = [n for n in range(1, 10_000_001)]

    for i in range(10_000_000):
        a[i] += b[i]

    print(f"last element = {a[-1]}")


if __name__ == "__main__":
    main()

File: sum_vectors.php
<?php

declare(strict_types=1);

ini_set('memory_limit', '-1');

$a = range(1, 10_000_000);
$b = range(1, 10_000_000);

for ($i = 0; $i < count($a); ++$i) {
    $a[$i] += $b[$i];
}

echo "last element = {$a[count($a) - 1]}";
